common/config/rush/pnpm-lock.yaml

```diff
@@ -1385,8 +1385,8 @@ packages:
       - encoding
     dev: false
 
-  /@azure/core-amqp@4.3.1:
-    resolution: {integrity: sha512-HfTAGhV1gLvq1FKk6yWi1sPhtnihbjquOrfc0RWYfqwY+CCPZGp4WhCFv4CC8AAdbTomKMLRJUP4bz0U/qa8iQ==}
+  /@azure/core-amqp@4.3.0:
```

Comment:
core-amqp is downgraded. Need to merge changes from latest main?

Reply:
addressed in https://github.com/Azure/azure-sdk-for-js/pull/30421/commits/d608d9699e334ef02ae795f753c691ab5de6a444